modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -292,6 +292,16 @@ export class Utils implements BaseUtils {
     return algosdk.encodeAddress(addr);
   }
 
+  /**
+   * decodeAddress return an addres decoding with algosdk
+   *
+   * @param addr
+   * @returns Uint8Array
+   */
+  decodeAddress(addr: string): any {
```

Comment:
try to return here Uinit8array not any

Reply:
It happens that it does not return a uint8array but an addres object that has a publicKey and the checksum